Customizing/global/plugins/Services/Repository/RepositoryObject/ReportStudyProgrammeOverview/classes/class.ilObjReportStudyProgrammeOverviewGUI.php

```diff
@@ -15,6 +16,7 @@ class ilObjReportStudyProgrammeOverviewGUI extends ilObjReportBaseGUI
 	protected static $success_img;
 	protected static $in_progress_img;
 	protected static $not_yet_started_img;
+	protected static $optional_img;
 
```

Comment:
Scusi hatte ich selber übersehen. Aber ich glaube hier in der Datei ist alles zu viel.